docs/source/querying_data/reranking.md

```diff
@@ -0,0 +1,119 @@
+# Reranking
+
+Second-stage reranking is a technique to use AI to gain significant search relevancy (we've seen +5-10% recall in the top 5) for your queries. OpenSearch 2.12 introduces the [Rerank processor](https://opensearch.org/docs/latest/search-plugins/search-pipelines/rerank-processor/), contributed by Aryn, to perform this operation. For more information, visit the [OpenSearch reranking documentation](https://opensearch.org/docs/latest/search-plugins/search-relevance/reranking-search-results/).
```

Comment:
For more information, visit the Opensearch reranking documentation and Search Pipelines documentation NEED LINK.